go.mod

```diff
@@ -4,9 +4,8 @@ go 1.12
 
 require (
 	github.com/gogo/googleapis v0.0.0-20180223154316-0cd9801be74a
-	github.com/gogo/protobuf v1.2.0
-	github.com/golang/protobuf v1.2.0
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6 // indirect
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.5.2
```

Comment:
I don't think we can introduce this version, it breaks compatibility with <v1.3.x. Can you revert this version change?